### PR TITLE
docs: keep examples tape names consistent

### PR DESCRIPTION
**Pull request by aymanbagabas**

### CODE REVIEW

File: examples/settings/set-font-family.tape [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Output examples/settings/font-family.gif
+Output examples/settings/set-font-family.gif
```
Comment (maaslalani) [inline examples/settings/set-font-family.tape:1]: I think this one is missing the font family (`Monoflow`)

**Review by maaslalani — APPROVED** [review]

**Comment by maaslalani** [comment]
Actually, fonts look a little off. Is JetBrains Mono installed?

**Comment by aymanbagabas** [comment]
> Actually, fonts look a little off. Is JetBrains Mono installed?

Ah I might not have it installed, my bad

**Comment by maaslalani** [comment]
Looking good thank you!

**Review by maaslalani — APPROVED** [review]

**Pull request merged**